examples/event-desc-multiple-descType.xml

```diff
@@ -6,17 +6,17 @@
       <p daptm:langSrc="original">
```

Comment:
shouldn't the langSrc attribute also be removed here?

Reply:
Yes, fixing

Reply:
Done in 932006b